agent/src/softwarecontaineragent.cpp

```diff
@@ -48,7 +48,6 @@ bool SoftwareContainerAgent::triggerPreload()
 inline bool SoftwareContainerAgent::isIdValid (ContainerID containerID)
 {
     return ((containerID < UINT32_MAX)
-            && (containerID >= 0)
```

Comment:
Yes but you can call function as IdValid(-1). 
Although in this case, it will return false due to map search, there is a meaning for this in my opinion. At least in this case, it will save us from searching map overhead.